docs/reference/team-processes/development-process.md

```diff
@@ -142,6 +142,68 @@ If a branch will survive beyond a couple of days its important to have a bit of
+
+It's generally too much overhead to re-review approved PRs with nitpick fixes just to make sure the fixup was done properly. We have "Dismiss stale pull request approvals when new commits are pushed" disabled in our branch protections for this reason.
+
+**Scenario (changes requested)**: A reviewer asks for changes on your PR. Depending on the breadth of changes required, you may want to commit those changes separately and regular push to your branch and when the reviewer approves, follow the fixup/force push steps above, and once CI passes again, you may merge the PR. Alternatively, you may rather go ahead fixup/force push to your branch so the reviewer can also see the final commit.
+
+**Scenario (draft PR)**: You have a branch containing several commits that you'd like to open as a draft for early feedback. You may fixup your commits at this stage if you wish which wouldn't require a force push, but it's fine for a PR to have multiple commits until you feel it's ready for review and merge. You might fill out the pull request title and template to help early reviewers and to give you something to copy and paste when you're ready to write the commit message. When your PR is ready for a review and potentially merge, follow the steps above to fixup your commits and force push.
+
+**Scenario (large PR)**: You've been working on a large task, which probably should have been broken down better, with many commits that may actually make the review process easier to keep them separate because they're loosely organized. It's fine to open a PR like this for review, but it's best to fill out the pull request title and template to help reviewers and to give you something to copy and paste when you're ready to write the commit message. Handle this how you and the reviewer feel is best, just be sure to fixup your commits before the PR is merged.
+
+### Merging a contributor's PR
+
+If someone outside of the fxa-devs group on Github makes a contribution, they will have to use their fork since they won't have write access to `mozilla/fxa`, see why this won't pass CI [on this issue][issue-forked-prs].
+
+Review the pull request per our "code review" guidelines below. When the PR appears ready, we will need to grab the contributor's commit and place it in a PR temporarily into `mozilla/fxa` marked as a draft and/or with a "do not merge" label or title. Circle CI will run the full suite based on that _commit_, meaning CI will appear to run on the contributor's branch.
+
+If you expect to only need to do this once, run:
+
+```bash
+git fetch git@github.com:<username>/fxa.git <commit hash>
+git checkout -b <whatever-branch-name> <commit hash>
+git push <remote> <whatever-branch-name> FETCH_HEAD # you can also use <commit hash> instead of FETCH_HEAD, as they're the same
```

Comment:
We don't need `FETCH_HEAD` here.

Reply:
Ahh you're right, thank you for catching. I somehow got the previous line mixed up with it.